examples/lang/docker0.mcl

```diff
@@ -0,0 +1,6 @@
+docker "mgmt-nginx" {
+    state => "running",
+    image => "nginx",
+    cmd => ["nginx", "-g", "daemon off;",],
```

Comment:
out of curiosity, why is there a semicolon?

Reply:
Copied directly from the nginx dockerfile. :P
https://github.com/nginxinc/docker-nginx/blob/f4d30145c60c433966df96f618d78513fee9d322/mainline/stretch/Dockerfile#L99